lib/workers/global/config/parse/file.spec.ts

```diff
@@ -71,32 +75,26 @@ describe('workers/global/config/parse/file', () => {
     ])(
       'fatal error and exit if error in parsing %s',
       async (fileName, fileContent) => {
-        const mockProcessExit = jest
-          .spyOn(process, 'exit')
-          .mockImplementationOnce(() => undefined as never);
+        processExitSpy.mockImplementationOnce(() => undefined as never);
         const configFile = upath.resolve(tmp.path, fileName);
         fs.writeFileSync(configFile, fileContent, { encoding: 'utf8' });
         await file.getConfig({ RENOVATE_CONFIG_FILE: configFile });
-        expect(mockProcessExit).toHaveBeenCalledWith(1);
+        expect(processExitSpy).toHaveBeenCalledWith(1);
         fs.unlinkSync(configFile);
       }
     );
 
     it('fatal error and exit if custom config file does not exist', async () => {
-      const mockProcessExit = jest
-        .spyOn(process, 'exit')
-        .mockImplementation(() => undefined as never);
+      processExitSpy.mockImplementation(() => undefined as never);
```

Comment:
```suggestion
      processExitSpy.mockImplementationOnce(() => undefined as never);
```

Reply:
`process.exit` is being called here twice, i think it is better to address this in a separate PR.

it seems one of the else branches is redundant.

 https://github.com/renovatebot/renovate/blob/728a8a1c685f684533cbcdfe9c751ebf264351e8/lib/workers/global/config/parse/file.ts#L41-L47
 
https://github.com/renovatebot/renovate/blob/728a8a1c685f684533cbcdfe9c751ebf264351e8/lib/workers/global/config/parse/file.ts#L66-L68